[docs][DOC-656] CDC: Add content for parallel streaming support using logical replication (#26254)
yugabyte/yugabyte-db: docs/content/preview/develop/change-data-capture/using-logical-replication/yugabytedb-connector.md

| `MaxQueueSizeInBytes` | long | The maximum buffer of the queue in bytes. This metric is available if `max.queue.size.in.bytes` is set to a positive long value. |
| `CurrentQueueSizeInBytes` | long | The current volume, in bytes, of records in the queue. |

## Advanced

> ### Using parallel streaming

Reviewer: ```suggestion
### Parallel streaming
```

> YugabyteDB also supports parallel streaming of a single table using logical replication - this essentially means that you can start the replication for the table using parallel tasks where each of them will be polling on specific tablets.

Reviewer: ```suggestion
{{<tags/feature/tp idea="1549">}}YugabyteDB also supports parallel streaming of a single table using logical replication. This means that you can start the replication for the table using parallel tasks, where each task polls on specific tablets.
```

{{< note title="Important" >}}

> To enable paralle streaming, you will need to enable the Technical Preview flags [ysql_enable_pg_export_snapshot](#) and [ysql_yb_enable_consistent_replication_from_hash_range](#).

Reviewer 2: parallel

{{< /note >}}

> Follow these steps to configure parallel streaming using the YugabyteDB Connector:

Reviewer: ```suggestion
Use the following steps to configure parallel streaming using the YugabyteDB Connector.
```

**Step 1: Decide on the number of tasks**

> This will be crucial as we will need to create the same number of replication slots and publications. Note that the number of tasks cannot be greater than the number of tablets you have in the table to be streamed.

Reviewer: ```suggestion
This is important, as you need to create the same number of replication slots and publications. Note that the number of tasks cannot be greater than the number of tablets you have in the table to be streamed.
```
